grammar.js

```diff
@@ -1135,7 +1134,7 @@ module.exports = grammar({
       ))
     )),
 
-    extension_expression: $ => prec('call', seq(
+    extension_expression: $ => prec.right(seq(
```

Comment:
This change is to simplify query injections.

Example:

```res
let _ = %raw(j`1 + 1`)
let _ = %raw(j`1 + 1`)
```

```
(source_file [0, 0] - [2, 0]
  (let_binding [0, 0] - [0, 22]
    (value_identifier [0, 4] - [0, 5])
    (extension_expression [0, 8] - [0, 22]
      (extension_identifier [0, 9] - [0, 12])
      (expression_statement [0, 13] - [0, 21]
        (template_string [0, 13] - [0, 21]
          (template_string_content [0, 15] - [0, 20])))))
  (let_binding [1, 0] - [1, 22]
    (value_identifier [1, 4] - [1, 5])
    (call_expression [1, 8] - [1, 22]
      function: (extension_expression [1, 8] - [1, 12]
        (extension_identifier [1, 9] - [1, 12]))
      arguments: (arguments [1, 12] - [1, 22]
        (template_string [1, 13] - [1, 21]
          (template_string_content [1, 15] - [1, 20]))))))
```

The first line is parsed as `extension_expression` and the second as `call_expression`.

Now, both have the same AST

```
(source_file [0, 0] - [2, 0]
  (let_binding [0, 0] - [0, 22]
    (value_identifier [0, 4] - [0, 5])
    (extension_expression [0, 8] - [0, 22]
      (extension_identifier [0, 9] - [0, 12])
      (expression_statement [0, 13] - [0, 21]
        (template_string [0, 13] - [0, 21]
          (template_string_content [0, 15] - [0, 20])))))
  (let_binding [1, 0] - [1, 22]
    (value_identifier [1, 4] - [1, 5])
    (extension_expression [1, 8] - [1, 22]
      (extension_identifier [1, 9] - [1, 12])
      (expression_statement [1, 13] - [1, 21]
        (template_string [1, 13] - [1, 21]
          (template_string_content [1, 15] - [1, 20]))))))
```

Reply:
This is fixed in #143